src/classes/STG_PanelADDRVerification_CTRL.cls

```diff
@@ -35,10 +35,47 @@
 */
 public with sharing class STG_PanelADDRVerification_CTRL extends STG_Panel {
 
+    /*******************************************************************************************************
+    * @description Interface for the address verifier
```

Comment:
Might be worth expanding the comment to explain how the interface is used.

Reply:
Done in the next commit, please let me know if/how the wording should be changed.